test/models/user_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
```

Comment:
Delete unused test files. Next time you can start a new Rails app without the default testing framework by using the `-T` flag `$rails new app -T`

[OPTIONAL]